src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+
+    private static final int NUMBER_OF_BALLS = 3;
+
+    public static void main(String[] args) {
+
+        Lottery[] lotteries = new Lottery[NUMBER_OF_BALLS];
```

Comment:
You only need one lottery object. When you call its getRandomBall() method, you will receive a new ball every time.

Reply:
+